android/fastlane/Fastfile

```
@@ -0,0 +1,80 @@
+        "Build Date" => Time.now.to_s,
+        "Build Number" => build_number,
+        "Internal App" => "<#{internal_app_url}>",
+      }
+    ) if ENV['CI']
+  end
+
+  lane :production do
+    upload_to_play_store(
+      track: "internal",
+      track_promote_to: "production",
+      skip_upload_changelogs: true,
```

RuboCop (1.73)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)